package/gluon-layer3-common/luasrc/usr/lib/lua/gluon/l3.lua

```diff
@@ -0,0 +1,21 @@
+local util = require("gluon.util")
+local site = require("gluon.site")
+
+local M = {}
+
+-- returns a prefix generated from the domain-seed
+-- for l3roamd -P <node-client-prefix>
+function M.node_client_prefix6()
+	local key = "gluon-l3roamd.node_client_prefix6"
```

Comment:
This key could include the current domain, for the client address to be unique per domain. Would that be useful?

Reply:
I think that's what should already be happening a few lines later, with the call to `util.domain_seed_bytes(key, 7)`. The key + the site's/domain's domain_seed will be the seeds to generate a prefix unique for this domain.

Check the other usages of domain_seed_bytes() in Gluon. We are using it like `...domain_seed_bytes("gluon-mesh-vxlan", 3)' to generate a VXLAN ID which is unique per domain, as well.

Reply:
Ah, perfect